src/lib/tusktask/mutators/mutateTaskData.ts

```
@@ -0,0 +1,29 @@
+      method: "PATCH",
+      headers: {
+        "content-type": "application/json",
+      },
+      body: JSON.stringify(req),
+    });
+
+    const data = await response.json();
+
+    return data;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)